modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementFailureTest.java

```diff
@@ -380,26 +381,77 @@ public void deleteOrganizations() {
     @Test(dependsOnMethods = "testUpdateDiscoveryAttributesUnauthorized")
     public void testGetPaginatedOrganizationsWithInvalidLimit() {
 
-        String invalidLimitUrl = ORGANIZATION_MANAGEMENT_API_BASE_PATH + QUESTION_MARK + LIMIT_QUERY_PARAM + EQUAL + "-1";
+        String invalidLimitUrl =
+                ORGANIZATION_MANAGEMENT_API_BASE_PATH + QUESTION_MARK + LIMIT_QUERY_PARAM + EQUAL + "-1";
         Response response = getResponseOfGetWithOAuth2(invalidLimitUrl, m2mToken);
         validateErrorResponse(response, HttpStatus.SC_BAD_REQUEST, ERROR_CODE_BAD_REQUEST);
     }
 
     @Test(dependsOnMethods = "testGetPaginatedOrganizationsWithInvalidLimit")
     public void testGetPaginatedOrganizationsWithInvalidAfterCursor() {
 
-        String invalidAfterCursorUrl = ORGANIZATION_MANAGEMENT_API_BASE_PATH + QUESTION_MARK + LIMIT_QUERY_PARAM + EQUAL + "10"
-                + AMPERSAND + AFTER_QUERY_PARAM + EQUAL + INVALID_CURSOR;
+        String invalidAfterCursorUrl =
+                ORGANIZATION_MANAGEMENT_API_BASE_PATH + QUESTION_MARK + LIMIT_QUERY_PARAM + EQUAL + "10"
+                        + AMPERSAND + AFTER_QUERY_PARAM + EQUAL + INVALID_CURSOR;
         Response response = getResponseOfGetWithOAuth2(invalidAfterCursorUrl, m2mToken);
         validateErrorResponse(response, HttpStatus.SC_BAD_REQUEST, ERROR_CODE_INVALID_PAGINATION_CURSOR);
     }
 
     @Test(dependsOnMethods = "testGetPaginatedOrganizationsWithInvalidAfterCursor")
     public void testGetPaginatedOrganizationsWithInvalidBeforeCursor() {
 
-        String invalidBeforeCursorUrl = ORGANIZATION_MANAGEMENT_API_BASE_PATH + QUESTION_MARK + LIMIT_QUERY_PARAM + EQUAL + "10"
-                + AMPERSAND + BEFORE_QUERY_PARAM + EQUAL + INVALID_CURSOR;
+        String invalidBeforeCursorUrl =
+                ORGANIZATION_MANAGEMENT_API_BASE_PATH + QUESTION_MARK + LIMIT_QUERY_PARAM + EQUAL + "10"
+                        + AMPERSAND + BEFORE_QUERY_PARAM + EQUAL + INVALID_CURSOR;
         Response response = getResponseOfGetWithOAuth2(invalidBeforeCursorUrl, m2mToken);
         validateErrorResponse(response, HttpStatus.SC_BAD_REQUEST, ERROR_CODE_INVALID_PAGINATION_CURSOR);
     }
+
+    @DataProvider(name = "organizationDiscoveryInvalidLimitAndOffsetDataProvider")
+    public Object[][] organizationDiscoveryInvalidLimitAndInvalidOffsetDataProvider() {
+
+        return new Object[][]{
+                {"0", "-1"}, {"5", "-1"}, {"20", "-1"}, {"25", "-1"}, {"", "-1"}, //invalid limit
```

Comment:
```suggestion
                {"0", "-1"}, {"5", "-1"}, {"20", "-1"}, {"25", "-1"}, {"", "-1"}, // Invalid limit.
```

Reply:
Comment should be in following format
1. First letter should be capital
2. Ends with full stop

Do the change in other places as well

Reply:
Addressed with commit: https://github.com/wso2/product-is/pull/21023/commits/7629f4c85aa7707952c2962016ec9bcd46e868a8